package.json

```diff
@@ -1,5 +1,11 @@
 {
   "name": "mifosx-community-app",
+  "description": "This is the default web application built on top of the MifosX platform for the mifos user community. It is a Single-Page App (SPA) written in web standard technologies like JavaScript, CSS and HTML5. It leverages common popular frameworks/libraries such as AngularJS, Bootstrap and Font Awesome.",
+  "author": {
+    "name": "openMF",
+    "email": "edcable@mifos.org",
```

Comment:
```suggestion 
    "email": "mifos-developer@lists.sourceforge.net",

Reply:
That's just edcable's email not the Mifos email.

Reply:
Thank you, I actually don't know their email address so I used their CEO's email. Thanks for commenting!